src/OrleansAzureUtils/Providers/Storage/AzureBlobStorage.cs

```diff
@@ -91,14 +91,21 @@ public class AzureBlobStorage : IStorageProvider
 
         IEnumerable<string> FormatPropertyMessage(IProviderConfiguration config)
         {
-            foreach (var property in new string[] { ContainerNamePropertyName, "SerializeTypeNames", "PreserveReferencesHandling", "UseFullAssemblyNames", "IndentJSON" })
+            var properties = new[]
+            {
+                ContainerNamePropertyName,
+                "SerializeTypeNames",
+                "PreserveReferencesHandling",
+                OrleansJsonSerializer.UseFullAssemblyNamesProperty,
```

Comment:
SerializeTypeNames and PreserveReferencesHandling are bogus, aren't they? we don't seem to be using them at all